inc/search.class.php

```diff
@@ -3994,8 +4202,10 @@ static function addDefaultWhere($itemtype) {
     * @param integer $meta         Is a meta search (meta=2 in search.class.php) (default 0)
     *
     * @return string Where string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta = 0) {
+   public function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta = 0) {
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.